github/repos_deployments_branch_policies.go

```diff
@@ -0,0 +1,119 @@
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+type DeploymentBranchPolicy struct {

```

Comment:
Please add "godoc"-style comments to each exported struct describing its purpose.